GameFiles/GameInterface/_integration_iw4x.gsc

```diff
@@ -330,19 +334,17 @@ NoClipImpl()
     else
     {
         self SetClientDvar( "sv_cheats", 1 );
```

Comment:
I recall this not being necessary since the server can change cheat-protected dvars regardless of sv_cheats' value.
This logic applies because it's the host sending the command to the client.